dts/arm/st/f4/stm32f405.dtsi

```diff
@@ -176,6 +176,40 @@
 			status = "disabled";
 			label= "OTGHS";
 		};
+
+		can1: can@40006400 {
+			compatible = "st,stm32-can";
+			#address-cells = <1>;
+			#size-cells = <0>;
+			reg = <0x40006400 0x400>;
+			interrupts = <19 0>, <20 0>, <21 0>, <22 0>;
+			interrupt-names = "tx", "rx0", "rx1", "sce";
```

Comment:
Any reason not using caps as done in L4 series?

Reply:
Hmm a quick grep for interrupt-names shows a mix of caps and non-caps, majority seems to be non-caps. 

But I guess it makes sense to have at least the CAN irq names all the same, so will change them to caps.